### PR TITLE
Letter rotated

### PR DESCRIPTION
**Pull request by vhaller**

### CODE REVIEW

File: src/components/exercises/LetterRotated.stories.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { storiesOf } from '@kadira/react-native-storybook';
+
+import LetterRotated from './LetterRotated';
+
+storiesOf('exercises/LetterRotated', module).add('', () => (
+  <LetterRotated
+    letters={['a', 'N', 'E', 's', 'T']}
+    rotated={4}
+    angle={'50deg'}
```
Comment (inyono) [inline src/components/exercises/LetterRotated.stories.js]: angle="50deg"

**Comment by inyono** [comment]
Note: Ignore the failing tests for now ^^ We forgot to update the Snapshot tests. I will do that (and add a Snapshot test for LetterRotated)

File: src/components/Components.js [hunk 2]
```diff
@@ -108,45 +109,100 @@ export const RoundButton = ({ icon, size, style, onPress }) => (
...
+    super(props);
+
+    this.state = {
+      size: new Animated.Value(props.size),
+      fontSize: new Animated.Value(this.getBaseFontSize(props))
+    };
+  }
+
+  getBaseFontSize = props =>
+    path(['textStyle', 'fontSize'], props) || DEFAULT.fontSize;
```
Comment (Knorrke) [inline src/components/Components.js]: just to clarify: path is used, because props.textStyle.fontSize fails when textStyle is undefined, yes?

Comment (inyono) [inline src/components/Components.js]: Exactly. Setting defaultProps would be another way to solve this.

**Comment by inyono** [comment]
Note /self: `Animated.parallel`, `interpolate`

**Pull request merged**